spec/references.bib

```diff
@@ -211,3 +226,18 @@ @online{hg-largefiles
   url="https://www.mercurial-scm.org/wiki/LargefilesExtension",
   urldate={2019-11-11}
 }
+
+@online{z-base32,
+    url="http://philzimmermann.com/docs/human-oriented-base-32-encoding.txt",
+    urldate={2020-08-18}
+}
+
+@online{whatwg-url,
+    url="https://url.spec.whatwg.org/commit-snapshots/1979119c2082b1e0f7f77aead4651700841a4abe/",
```

Comment:
Should I be worried if I got to this link and it says:
> Do not attempt to implement this version of the standard. Do not reference this version as authoritative in any way. 

:grimacing:

Reply:
Haha, the WHATWG is so semantic web that they don't provide bibtex references. It's "living standards", you know?

I'm not sure if I should just link to https://url.spec.whatwg.org (because, I don't even know what the `url` crate conforms to), or rather submit an RFC to the w3c to replace all HTML + JSON on the web with LaTeX.

Reply:
I guess we'll live with their standards